tests/codegen/map_increment_decrement.cpp

```diff
@@ -0,0 +1,168 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, map_increment_decrement)
+{
+  test("BEGIN { @x = 10; @x++; ++@x; @x--; --@x; }",
```

Comment:
Could also test undefined map here:

```suggestion
  test("BEGIN { @x = 10; @x++; ++@x; @x--; --@x; @y++ }",
```